doc/builders/images/dockertools.xml

```diff
@@ -12,7 +12,7 @@
   <title>buildImage</title>
 
   <para>
-   This function is analogous to the <command>docker build</command> command, in that it can be used to build a Docker-compatible repository tarball containing a single image with one or multiple layers. As such, the result is suitable for being loaded in Docker with <command>docker load</command>.
+   This function is analogous to the <command>docker build</command> command, in that it can be used to build a Docker-compatible repository tarball containing a single image with one or multiple layers. As such, the <literal>*.tar.gz</literal> file of the <i>result</i> is suitable for being loaded into the (local) registry with <command>docker load</command>.
```

Comment:
In the interest of precise wording, it's not an actual registry locally, and loading the `tar.gz` is a different operation that what happens when the image is pushed to an actual registry.

If you use docker tooling, the `docker import ...tar.gz` is separate from the step to push to a registry.

(And in case it's useful to you to know, you can use other tooling to push the image to a registry without having to first import it to the local docker store, might be worth documenting as a separate PR. If you're interested, see here: https://github.com/colemickens/flake-azure/blob/dev/docker/azure-aio/default.nix#L31)

For here, I might just say something like (also changed the wording since the docker instance imported to can vary based on env vars, etc):

```
is suitable for being loaded into a docker instance via <command>docker load ./result</command>.
```

Reply:
Right it's just the _store_. I was trying to mimic that docker (aren't those oci?) tars - as you mention - can be loaded directly to any registry compliant interface.

Reply:
The point I was trying to make is that the tars can *not* be directly loaded to a Docker registry... They have to either be imported or processed and then uploaded by a tool like skopeo. (to put this another way, you can *not* `docker push /some/tar.gz docker.io/some/image`).

Importing to the store and pushing/importing to a registry are different things.

Reply:
The docs infra is daunting. It is [DocBook](https://docbook.org/docs/), not HTML. As such, some tags that *obviously should work* simply don't exist.

The current change would break the manual build.

```
/build/doc/manual-full.xml:12129:261: error: element "i" not allowed anywhere; 
```

`<para>` can [have the children as documented here](https://tdg.docbook.org/tdg/5.1/para.html#element.db.para.6.6). I'm not sure which is better, possibly `<emphasis>` is the one to use.

Reply:
> Importing to the store and pushing/importing to a registry are different things.

The new wording after the amend is _at least not wrong_. Would you agree?